ifrs17-template/Constants/CalculationEngine.ipynb

```diff
@@ -22,7 +22,8 @@
                 "var projectName = \"ifrs17\";",
                 "\nvar environmentName = \"v1.3.0\";",
                 "\nvar notebookName = \"CalculationEngine\";",
-                "\nvar calculationEngine = $\"#!import \\\"//{projectName}/{environmentName}/{notebookName}\\\"\";"
+                "\nvar calculationEngine = $\"#!import \\\"//{projectName}/{environmentName}/{notebookName}\\\"\";",
```

Comment:
TO RESET

Reply:
@davide does this need a reset, or its alredy fine ?

Reply:
it is fine as is. Resolving the conversation.